text/0000-rfc-stages.md

```diff
@@ -0,0 +1,189 @@
+
+Ember's users should be able to look at an RFC and know more about how close it is to being part of a stable release. This proposal introduces explicit stages for RFCs, covering the steps from the initial draft to the shipped result. Inspired by TC39, these stages are a communication and collaboration tool tool. They can give the community greater visibility into Ember's development, encourage participation, and improve cross-team coordination. This RFC does not aim to substantially change the existing process, but rather apply labels to what already happens informally.
+
+## Motivation
+
+It can be difficult for both users and contributors of Ember to know the status of a new feature. Has it been approved? Is it a work in progress? What still needs to be done? Which version of Ember did it ship in?
+
+We can see that stages are needed when we consider the following:
+
+1. Community feedback that it was difficult to follow along with the progression of Ember Octane's features
+2. Challenges uncovered in the existing RFC Tracking process
+3. The long-standing, but stagnant goal of providing a public-facing status board
+4. Confusion about what it means for an RFC to be merged. There is a common myth that a merged RFC indicates that a feature will be in the next release, when instead it is a green flag for implementation work to begin.
+
+To understand the problem, it is helpful to think of the informal stages a merged RFC already can progress through: We have RFCs for which we have needed to indicate a change in direction ([MU](https://emberjs.github.io/rfcs/0143-module-unification.html)), RFCs that have needed to be clarified/replaced ([Named Blocks](https://emberjs.github.io/rfcs/0226-named-blocks.html)), RFCs waiting to be implemented, and RFCs whose features have been included in a stable release. Each of these scenarios was handled in a thoughtful manner, but they were difficult for users to follow along with.
+
+This proposal aims to build on the success of RFC 300, [RFC (Request for Comments) Process Update]([https://github.com/emberjs/rfcs/blob/master/text/0300-rfc-process-update.md](https://github.com/emberjs/rfcs/blob/master/text/0300-rfc-process-update.md)).
+
+## Detailed design
+
+### Stages:
+
+A successful RFC can move through the following stages:
+
+| Stage| Description| Criteria to advance to next step |
+| -----| -----------|----------------------------------| 
+| Proposed | A pull request made to the RFC repository which is open for public comment and core team evaluation. | "The RFC has been in Final Comment Period, and the relevant core team has consensus to merge it, thus accepting the proposal." |
+| Accepted | "The work to integrate the proposal into Ember's codebases or materials may begin. An ""Accepted"" RFC is not necessary in-progress." | "The technical implementation is complete according to plan outlined in the RFC, and is in harmony with any changes in Ember that have occurred since the RFC was first written." |
```

Comment:
Realistically, an RFC can be implemented even before it is opened. It's not clear that "in-progress" is a stage rather than a status that can be concurrent with any of the stages before "Released". This RFC was just updated with more detail, the current implementation status should be clear in the PR for "Ready for Release" for the RFC.

Reply:
@kategengler yea you make a good point. I just re-read the RFC with the most recent updates and I'm happy that the "status" of the RFC can be represented effectively 👍